blip-tap.md

```diff
@@ -924,21 +924,26 @@ accept if favorable
 
 ##### Request For Quote (`tap_rfq`)
 
-When a receiver wishes to receive `N` units of TAP asset ID `asset_id`, a new
+When a receiver wishes to receive `N` units of TAP asset ID `in_asset_id`, a new
```

Comment:
Term here assumes only receiver, but now can be receiver/sender based on the new fields below.